src/main/java/org/jabref/pdfimport/PdfImporter.java

```diff
@@ -29,7 +29,7 @@
 import org.jabref.logic.l10n.Localization;
 import org.jabref.logic.util.UpdateField;
 import org.jabref.logic.util.io.FileUtil;
-import org.jabref.logic.xmp.XMPUtil;
+import org.jabref.logic.xmp.XMPUtilShared;
```

Comment:
Maybe rename the class to `XmpUtilShared` to follow Google's Casing rules? (Similar for method names, ...)